numpy/random/randomgen/examples/cython/extending.pyx

```diff
@@ -1,13 +1,17 @@
+#cython: language_level=3
+from libc.stdint cimport uint32_t
+from cpython.pycapsule cimport PyCapsule_IsValid, PyCapsule_GetPointer
+
 import numpy as np
 cimport numpy as np
 cimport cython
-from libc.stdint cimport uint32_t
-from cpython.pycapsule cimport PyCapsule_IsValid, PyCapsule_GetPointer
+
 from randomgen.common cimport brng_t
 from randomgen.xoroshiro128 import Xoroshiro128
```

Comment:
I think this needs to be "from numpy.random.randomgen

Reply:
Fixed now.